src/bin/wrapped-wit.js

```diff
@@ -0,0 +1,1029 @@
+        options: [
+          'port',
+        ],
+        envars: [],
+      },
+      supplies: {
+        hint: `Show relevant token-related supplies on ${helpers.colors.mcyan(ethRpcNetwork.toUpperCase())}.`,
+        options: [
+          'witnet',
+          'port',
+          ...(WrappedWIT.isNetworkCanonical(ethRpcNetwork)
+            ? [ 'witnet', ]
+            : []
+          ),
+        ],
+        envars: [
+          ...(WrappedWIT.isNetworkCanonical(ethRpcNetwork)
+            ? [ 'WITNET_SDK_PROVIDER_URL', 'WITNET_SDK_WALLET_MASTER_KEY', ]
+            : []
+          ),
+        ],
+      },
+      transfers: {
+        hint: `Show transfers of Wrapped/WIT tokens on ${helpers.colors.mcyan(ethRpcNetwork.toUpperCase())}.`,
+        flags: [
+          'burns',
+          'mints',
+        ],
+        options: [
+          'port',
+          'limit',
+          'offset',
+          'since',
+          'from',
+          'into',
+          'value',
+          'gasPrice',
+        ],
+      },
+    } : {}),
+    ...(WrappedWIT.isNetworkCanonical(ethRpcNetwork) ? {
+      wrappings: {
+        hint: `Show wrapping transactions into ${helpers.colors.mcyan(ethRpcNetwork.toUpperCase())}.`,
+        flags: [
+          'check',
+        ],
+        options: [
+          'witnet',
+          'port',
+          'limit',
+          'offset',
+          'since',
+          'from',
+          'into',
+          'value',
+          'vtt-hash',
+          'gasPrice',
+        ],
+        envars: [
+          'WITNET_SDK_PROVIDER_URL',
+          'WITNET_SDK_WALLET_MASTER_KEY',
+        ],
+      },
+      unwrappings: {
+        hint: `Show unwrapping transactions from ${helpers.colors.mcyan(ethRpcNetwork.toUpperCase())}.`,
+        flags: [
+          'check',
+        ],
+        options: [
+          'witnet',
+          'port',
+          'limit',
+          'offset',
+          'since',
+          'from',
+          'into',
+          'value',
+          'gasPrice',
+        ],
+        envars: [
+          'WITNET_SDK_PROVIDER_URL',
+        ],
+      },
+    } : {}),
+    gateway: {
+      hint: "Launch a local ETH/RPC signing gateway connected to some specific EVM network.",
+      params: ["EVM_NETWORK"],
+      options: [
+        'port',
+      ],
+      envars: [
+        'ETHRPC_PRIVATE_KEYS',
+        'ETHRPC_PROVIDER_URL',
+      ],
+    },
+    networks: {
+      hint: "List EVM networks where the Wrapped/WIT token is available.",
+      params: "[EVM_ECOSYSTEM]",
+      flags: [
+        'testnets',
+      ],
+    },
+    commands: {
+      address,
+      balance,
+      gateway,
+      networks,
+      supplies,
+      transfers,
+      unwrappings,
+      wrappings,
+    },
+  }
+
+  var [args, flags] = helpers.extractFlagsFromArgs(process.argv.slice(2), Object.keys(settings.flags))
```

Comment:
We should never use `var`. It's really error prone, can make weird scope errors and force us to think how variables can be `hoisted` in different ways. There is no reason to use it nowadays